server_addon/houdini/server/settings/create.py

```diff
@@ -170,7 +180,8 @@ class CreatePluginsModel(BaseSettingsModel):
     },
     "CreateUSDRender": {
         "enabled": False,
```

Comment:
With this PR do we now want to update the `CreateUSD` and `CreateUSDRender` creator enabled state to `True`?

Reply:
Good question. 
I might not have a full answer, but here is what I think. 

I believe we might want to keep it as experimental for now, and when we move out the AyonUsd plugin as a Beta or Alpha setup, we then go and say, if you activate AyonUsd, all the Usd stuff will be activated too. maybe we also want to create the logic that Uses URI instead of the path for Ayon items so that we don't risk fragmenting the USD user base. 

I know that's not an answer at all, but maybe some brain food.

Reply:
what is the MVP for `CreateUSD` and `CreateUSDRender`  to be considered as not experimental ?

Reply:
Still a solid question. @antirotor what would you like to do here?

Reply:
I've opted for now with https://github.com/ynput/ayon-core/pull/295/commits/5c0ad8512981b5d4b621737bf538f18d3359604c to enable by default since I feel like that's the goal with this PR and what most people will seek after.